spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java

```diff
@@ -167,6 +168,9 @@ public static WriteMethod from(@Nullable String writeMethod) {
 
   public static final String GPN_ATTRIBUTION = "GPN";
 
+  public static final String BIG_NUMERIC_DEFAULT_PRECISION = "bigNumericDefaultPrecision";
```

Comment:
The parameter names should match the documentation (and exceptions)

Reply:
Done